math/OpenBLAS/Portfile

```diff
@@ -22,22 +22,9 @@ platforms           darwin
 compiler.blacklist-append cc {*gcc-3*} {*gcc-4.[0-7]} {clang < 800.0.38}
 platform darwin i386 {
     compiler.blacklist-append {macports-clang-[3-4].*}
-    compiler.fallback-append macports-clang-5.0 macports-clang-6.0 macports-clang-7.0
+    compiler.fallback-append macports-clang-9.0 macports-clang-8.0 macports-clang-7.0
```

Comment:
Is this line still needed ? These compilers should be in the fallback anyway by default now.

Reply:
After we get this running, you are right, there is some more cleanup to be done in the Portfile -- I just made a tiny change here. But the new defaulting that exists since this was originally put in (way back when) does indeed make these fallbacks no longer needed.

Reply:
Fixed!